sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/helpers/ConsoleLogger.java

```diff
@@ -7,21 +7,27 @@
     private static final String ANSI_RED = "\u001B[31m";
     private static final String ANSI_GREEN = "\u001B[32m";
     public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_BLUE = "\u001B[34m";
     private static final String ANSI_CYAN = "\u001B[36m";
 
-    public static void PrintHeader(String message) {
+    public static void printHeader(String message) {
         System.out.println(ANSI_CYAN + "\n===" + message.toUpperCase() + "===\n" + ANSI_RESET);
     }
 
-    public static void PrintSuccess(String message) {
-        System.out.println(ANSI_GREEN + message + ANSI_RESET);
+    public static void print(String message) {
```

Comment:
Could you add some comments to these helper methods, describing their user-cases. 
Also, just FYI - Jonathan recommended that we don't use println, I have asked for clarification on if we should use printf or logger.